platform/qt/src/qmapboxgl.cpp

```diff
@@ -161,12 +155,41 @@ std::unique_ptr<const mbgl::SpriteImage> toSpriteImage(const QImage &sprite) {
 */
 
 /*!
+    \enum QMapboxGLSettings::ConstrainMode
+
+    This enum sets whenever a map should wrap.
```

Comment:
Unsure about the language here: what do you mean by "sets"?  Does it set itself or does the user have to set it? Like above, can change to `Determines if the map wraps` or similar.